lib/modules/commandLine.js

```diff
@@ -84,16 +84,19 @@ const commandLine = _.once(() => {
 		}
 	});
 
-	input.addEventListener('keyup', (e: KeyboardEvent) => {
+	document.addEventListener('keyup', (e: KeyboardEvent) => {
 		if (e.key === NAMED_KEYS.Escape) {
 			// close prompt.
 			toggleCmdLine(false);
-		} else {
-			// auto suggest?
-			cmdLineHelper($(e.target).val());
+			e.stopImmediatePropagation();
```

Comment:
Hmmm, this listener never gets removed from the document, do we want to call this for all keypresses?